cmd/exchange_wrapper_issues/main.go

```diff
@@ -0,0 +1,838 @@
+			IntermediaryBankName:          config.BankDetails.IntermediaryBankName,
+			IntermediaryBankAccountNumber: config.BankDetails.IntermediaryBankAccountNumber,
+			IntermediarySwiftCode:         config.BankDetails.IntermediarySwiftCode,
+			IntermediaryIBAN:              config.BankDetails.IntermediaryIban,
+			IntermediaryBankCity:          config.BankDetails.IntermediaryBankCity,
+			IntermediaryBankAddress:       config.BankDetails.IntermediaryBankAddress,
+			IntermediaryBankCountry:       config.BankDetails.IntermediaryBankCountry,
+			IntermediaryBankPostalCode:    config.BankDetails.IntermediaryBankPostalCode,
+			IntermediaryBankCode:          config.BankDetails.IntermediaryBankCode,
+		}
+		var r22 string
+		r22, err = e.WithdrawFiatFunds(&fiatWithdrawRequest)
+		msg = ""
+		if err != nil {
+			msg = err.Error()
+			responseContainer.ErrorCount++
+		}
+		responseContainer.EndpointResponses = append(responseContainer.EndpointResponses, EndpointResponse{
+			SentParams: jsonifyInterface([]interface{}{fiatWithdrawRequest}),
+			Function:   "WithdrawFiatFunds",
+			Error:      msg,
+			Response:   r22,
+		})
+
+		var r23 string
+		r23, err = e.WithdrawFiatFundsToInternationalBank(&fiatWithdrawRequest)
+		msg = ""
+		if err != nil {
+			msg = err.Error()
+			responseContainer.ErrorCount++
+		}
+		responseContainer.EndpointResponses = append(responseContainer.EndpointResponses, EndpointResponse{
+			SentParams: jsonifyInterface([]interface{}{fiatWithdrawRequest}),
+			Function:   "WithdrawFiatFundsToInternationalBank",
+			Error:      msg,
+			Response:   r23,
+		})
+		response = append(response, responseContainer)
+	}
+	return response
+}
+
+func jsonifyInterface(params []interface{}) json.RawMessage {
+	response, _ := json.MarshalIndent(params, "", " ")
+	return response
+}
+
+func loadConfig() (Config, error) {
+	var config Config
+	file, err := os.OpenFile("wrapperconfig.json", os.O_RDONLY, os.ModePerm)
+	if err != nil {
+		return config, err
+	}
+	defer file.Close()
+	keys, err := ioutil.ReadAll(file)
+	if err != nil {
+		return config, err
+	}
+
+	return config, common.JSONDecode(keys, &config)
+}
+
+func saveConfig(config *Config) {
+	log.Println("JSONifying config...")
+	jsonOutput, err := json.MarshalIndent(config, "", " ")
+	if err != nil {
+		log.Fatalf("Encountered error encoding JSON: %v", err)
+		return
+	}
+
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Printf("Encountered error retrieving output directory: %v", err)
+		return
+	}
+
+	log.Printf("Outputting to: %v", filepath.Join(dir, "wrapperconfig.json"))
+	err = common.WriteFile(filepath.Join(dir, "wrapperconfig.json"), jsonOutput)
+	if err != nil {
+		log.Printf("Encountered error writing to disk: %v", err)
+		return
+	}
+}
+
+func outputToJSON(exchangeResponses []ExchangeResponses) {
+	log.Println("JSONifying results...")
+	jsonOutput, err := json.MarshalIndent(exchangeResponses, "", " ")
+	if err != nil {
+		log.Fatalf("Encountered error encoding JSON: %v", err)
```

Comment:
This will call os.Exit(1) so there's no need to return, unless you meant Printf here?